hivemind/optim/experimental/state_averager.py

```diff
@@ -378,20 +378,31 @@ def step(
                     self.finished_optimizer_step.clear()
         return output
 
-    def _do(self, optimizer_step: bool, zero_grad: bool, averaging_round: bool, **kwargs):
+    def _do(self, optimizer_step: bool, zero_grad: bool, averaging_round: bool, grad_scaler: Optional[GradScaler], **kwargs):
```

Comment:
_step_inner?

Reply:
Let's discuss this with @borzunov on the main PR